docs/web-code-style.md

```diff
@@ -0,0 +1,20 @@
+<!--
+   - SPDX-FileCopyrightText: 2021 Serokell <https://serokell.io>
+   -
+   - SPDX-License-Identifier: CC0-1.0
+   -->
+
+As foundation for our TypeScript/SCSS styleguide
+[this styleguide](!https://basarat.gitbook.io/typescript/styleguide) is used,
```

Comment:
Why does it have this `!`? Does it work at all? Apparently it doesn't.

Reply:
it worked in Idea, I will remove

Reply:
The link works now.